### DISCUSSION TITLE
亮度功能及相关建议

### DISCUSSION THREAD

**Discussion by Yumeyo23** [shuax/MouseInc.Settings#95] [Q&A]
非常喜欢MouseInc软件，帮我解决了多年用国产浏览器，转到Chrome后鼠标手势问题，还有超好用的全局手势。特别来注册一个账号，希望提点自己的建议~感谢阅读。

在最近的亮度调整功能中，软件可直接控制笔记本显示器亮度了，之前是类似于一层黑色覆盖的效果，不控制显示器亮度(不知道是不是个例？但是在我的笔记本上是这样的)。
改版后的效果也不错，但个人感觉改版后，调亮度过程变卡了，滚轮明显不如之前顺畅
当然也可能是我电脑配置不太好（8代i5）？看到其他朋友也在反馈亮度功能，个人感觉应该不是个例
此外，我觉得之前的亮度调整模式(2.11.2之前的版本)挺好的，平滑不卡，而且更喜欢这种调整思路，如果有OLED屏的设备，这样调亮度说不定能避免低亮度PWM调光问题:)
希望能在后续版本中能让用户自选使用哪种亮度调整方案~
对了作者大大，如果有拿捏不定的选项，我觉得可以都留在设置里，用户自己选择是否需要开启，也许更能满足不同用户的需要~
当然如果这样很麻烦，那就还是按作者方便的来吧！

再次感谢作者大大的软件，希望软件越做越好！望回复，谢谢 awa~

**Comment by shuax** [comment]
可能会加个调节模式
